mock/server/worker.go

```diff
@@ -199,3 +206,22 @@ func registerToken(c *gin.Context) {
 
 	c.JSON(http.StatusCreated, body)
 }
+
+// getQueueCreds returns mock JSON for a http POST.
+func getQueueCreds(c *gin.Context) {
+	q := c.Param("queue")
```

Comment:
I don't think this is possible since the URL path doesn't have a queue parameter

Reply:
Yea it's not possible! I tried! You have any suggestions here?

Reply:
I would mock a failed attempt by having a user pass in no token or something and then expect an error

Reply:
then would be in sdk-go that is using the mock. getQueueCreds has a mustRegToken perm attached.